classes/class-pods-beaver-page-data.php

```diff
@@ -555,13 +559,14 @@ public static function pods_get_settings_fields( $field_options = array() ) {
 	 * @param string $pod_name
 	 * @param array  $field_options Field options based on pod_data array.
 	 * @param string $prefix
-	 * @param array  $pods_visited
+	 * @param array  $pods_fields_visited
 	 *
 	 * @return array
+	 * @internal param array $pods_visited
 	 *
 	 * @since 1.0
 	 */
-	private static function recurse_pod_fields( $pod_name, $field_options = array(), $prefix = '', &$pods_visited = array() ) {
+	private static function recurse_pod_fields( $pod_name, $field_options = array(), $prefix = '', &$pods_fields_visited = array() ) {
```

Comment:
Why not use a `static $pods_fields_visited = array();` in this method instead of passing it around?